hijack/helpers.py

```diff
@@ -47,16 +47,22 @@ def release_hijack(request):
     if not hijack_history:
         raise PermissionDenied
 
-    hijacker = None
-    hijacked = None
-    if hijack_history:
-        hijacked = request.user
-        user_pk = hijack_history.pop()
-        hijacker = get_object_or_404(get_user_model(), pk=user_pk)
-        backend = get_used_backend(request)
-        hijacker.backend = "%s.%s" % (backend.__module__, backend.__class__.__name__)
-        with no_update_last_login():
-            login(request, hijacker)
+    hijacked = request.user
+    user_pk = hijack_history.pop()
+    hijacker = get_object_or_404(get_user_model(), pk=user_pk)
+    backend = get_used_backend(request)
+    hijacker.backend = "%s.%s" % (backend.__module__, backend.__class__.__name__)
+
+    pre_hijack_ended_results = pre_hijack_ended.send(
```

Comment:
Please see my comments below.
```suggestion
    pre_hijack_ended.send(
```